src/circuit/gadget/sinsemilla/chip.rs

```diff
@@ -110,15 +105,33 @@ impl SinsemillaChip {
         config.generator_table.load(layouter)
     }
 
+    /// # Side-effects
+    ///
+    /// All columns in `advices` and `constants` will be equality-enabled.
     #[allow(clippy::too_many_arguments)]
     #[allow(non_snake_case)]
     pub fn configure(
         meta: &mut ConstraintSystem<pallas::Base>,
         advices: [Column<Advice>; 5],
         lookup: (Column<Fixed>, Column<Fixed>, Column<Fixed>),
         constants: [Column<Fixed>; 6], // TODO: replace with public inputs API
-        perm: Permutation,
     ) -> <Self as Chip<pallas::Base>>::Config {
+        // This chip requires all advice columns and the `constants` fixed column to be
+        // equality-enabled. The advice columns are equality-enabled by the calls to
+        // LookupRangeCheckConfig::configure.
```

Comment:
```suggestion
        // This chip requires all advice columns and the `constants` fixed columns to
        // be equality-enabled. These columns are equality-enabled by the calls to
        // LookupRangeCheckConfig::configure.
```

Reply:
Ah whoops, here I was referring to the `constants` parameter of `SinsemillaConfig`, not the `constants` parameter of `SinsemillaConfig::configure` 😆